src/gui/shutdownconfirmdlg.cpp

```diff
@@ -30,20 +30,22 @@
  * Contact : hammered999@gmail.com
  */
 
-#include <QVBoxLayout>
-#include <QHBoxLayout>
+#include "shutdownconfirmdlg.h"
+#include "ui_shutdownconfirmdlg.h"
```

Comment:
I agree. And numbers 2+3 should have the headers sorted by name(each number separately of course).